docset/windows/iisadministration/new-iissitebinding.md

```diff
@@ -14,7 +14,7 @@ keywords: powershell, cmdlet
 # New-IISSiteBinding
 
 ## SYNOPSIS
-Adds a new binding to an existing Website.
+Adds a new binding to an existing Website. This cmdlet has been introduced in version 1.1.0.0 of IISAdministration module.
```

Comment:
Adds a new binding to an existing **w**ebsite

Reply:
@get-itips do you agree with Nicole's suggestion?

Reply:
corrected.